src/EFCore.PG.NodaTime/Infrastructure/Internal/NpgsqlNodaTimeOptionsExtension.cs

```diff
@@ -23,6 +23,10 @@ public virtual bool ApplyServices(IServiceCollection services)
 
         public virtual long GetServiceProviderHashCode() => 0;
 
+        public void PopulateDebugInfo(IDictionary<string, string> debugInfo)
```

Comment:
Seems like we should follow SQL Server/Sqlite here and populate like them (see their own corresponding files for an example).